.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "autorest.common"]
```

Comment:
Make sure you clone with `--recurse-submodules`  or init the submodule after pulling...